src/cpu/x64/gemm/gemm_info.cpp

```diff
@@ -366,8 +366,8 @@ void gemm_info_t<a_t, b_t, c_t>::jit_init(void) {
         bool is_bf16_amx = is_bf16 && mayiuse(avx512_core_bf16_amx_bf16);
         bool is_amx = is_int8_amx || is_bf16_amx;
 
-        static std::unique_ptr<jit_generator> copy_a[2][2] = {{nullptr}};
-        static std::unique_ptr<jit_generator> copy_b[2][2] = {{nullptr}};
+        static maybe_unique_ptr<jit_generator> copy_a[2][2] = {{nullptr}};
+        static maybe_unique_ptr<jit_generator> copy_b[2][2] = {{nullptr}};
```

Comment:
It's not directly related to this PR but can you please remove redundant initialization to `nullptr`?